### ISSUE TITLE
v9 migration sprint

### ISSUE THREAD

**Issue by rvalyi** [OCA/l10n-brazil#184]
In order to not port twice the same modules, please comment here if you are going to port a module and tell us about the PR you created.

Here are the module for which you should start from the 8.0 OCA branch:
- [ ] l10n_br_account
- [ ] l10n_br_base
- [ ] l10n_br_crm
- [ ] l10n_br_crm_zip
- [ ] l10n_br_data_account
- [ ] l10n_br_data_base
- [ ] l10n_br_zip

As for the other modules, most of them are already migrated in branches which PR's are awaiting fixes, tests and review to meet the OCA quality standards. For these PR's you should start from the work in progress PR:
- [ ] l10n_br_account_product
- [ ] l10n_br_stock
- [ ] l10n_br_sale_stock
- [ ] l10n_br_sale_service
- [ ] l10n_br_sale_product
- [ ] l10n_br_sale
- [ ] l10n_br_purchase
- [ ] l10n_br_timesheet_invoice
- [ ] l10n_br_delivery
- [ ] l10n_br_data_account_service
- [ ] l10n_br_data_account_product
- [ ] l10n_br_account_voucher
- [ ] l10n_br_account_service
- [ ] l10n_br_account_product_service
- [ ] l10n_br_account_product

**Comment by mileo** [comment]
Can you please create a milestone?

**Comment by rvalyi** [comment]
@mileo the milestone feature isn't used widely in the OCA, but why not.

Now my doubt is: what would be its scope? It's very likely it will take like more than 12 months to migrate 100% of the modules due to the numerous accounting changes and low project resources (it's kind of very common on the other OCA repo too and it's also important for the sustainability of the model that users/customers get the message that there is no free lunch and that the work is demand driven). So would it make any sense to have a milestone to migrate 100% of the modules?

if you agree, what we can do is have a 1st milestone to migrate modules that don't depend on the account module initially. This one I think we can get it migrated in less than 1 or 2 months. And then we would plane the next one.

What do you think?

**Comment by danimaribeiro** [comment]
I think we can do better this time.
4 months to migrate everything seems reasonable, I know it's hard, but is not rocket science.

https://github.com/Trust-Code/l10n-brazil/tree/9.0
Stuck with account, but the other ones are already working, so 1 day to migrate the ones who don't depend on account changes. There is no need to have a PR open for a month.

**Comment by rvalyi** [comment]
@danimaribeiro as I explained I totally doubt upstream accounting issues will be fixes so fast at the OCA level. Because changes are so numerous, because even if the code sucks much less it's totally not battle tested yet (not even at Odoo SA yet, no customer have been migrated yet either) and also because large OCA players who will rule how upstream will behave are very unlikely to have critical v9 projects funding so much work in a so narrow time frame given what I explained. But bet taken.

So let's do a 1st milestones with the non accounting modules anyway? Everybody OK with that?
